data/src/main/java/org/oppia/android/data/backends/gae/model/GaeFeedbackReport.kt

```diff
@@ -0,0 +1,20 @@
+package org.oppia.android.data.backends.gae.model
+
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+
+/**
+ * Data class for the feedback report sent by the Android app to remote storage.
+ * TODO(#2801): Link backend domain model
+ */
+@JsonClass(generateAdapter = true)
+data class GaeFeedbackReport constructor(
+
+  @Json(name = "api_key") val apiKey: String,
+  @Json(name = "report_creation_timestamp_ms") val reportCreationTimestampMs: Double?,
+  @Json(name = "user_supplied_feedback") val userSuppliedFeedback: GaeUserSuppliedFeedback?,
```

Comment:
Here & elsewhere: why are these properties optional (e.g. nullable)? Suggest actually making the optional ones nullable and everything else should be required.

Reply:
Ah I presumed that this was a nuance of data classes since all other GAE models I saw had nullable fields, will change that.